charts/stable/openproject/values.yaml

```diff
@@ -39,5 +39,8 @@ postgresql:
   postgresqlUsername: openproject
   postgresqlDatabase: openproject
 
+memcached:
+  enabled: true
+
```

Comment:
I'm not sure if this is enough.
Doesn't it need to connect them somehow?

A URL for example..
@Ornias1993

Reply:
yes